roles/bootstrap/tasks/main.yml

```diff
@@ -103,6 +109,28 @@
   shell: /bin/chown -R {{ item }}:{{ item }} {{ notebooks_folder }}/{{ item }}
   with_items: "{{ users }}"
 
+- name: Get dataset
+  become: true
+  become_user: root
+  unarchive:
+    src: https://storage.googleapis.com/gdd-trainings-bucket/ds-spark-data.zip
+    remote_src: yes
+    dest: "{{ notebooks_tmp_folder }}/data"
+    creates: "{{ notebooks_tmp_folder }}/data/band1.txt"
+  when: include_data
+
+- name: Create data dir in HDFS filesystem
+  become: true
+  become_user: root
```

Comment:
shouldn't you become the `hdfs` user?

Reply:
Root user extracted the data. Dataproc cluster is not secured so no need to become hdfs user.